vmas/simulator/environment/gymnasium.py

```diff
@@ -0,0 +1,183 @@
+#  Copyright (c) 2022-2024.
+#  ProrokLab (https://www.proroklab.org/)
+#  All rights reserved.
+from typing import List, Optional
+
+import gym
+import gymnasium
```

Comment:
```suggestion
_has_gymnasium = importlib.util.find_spec("gymnasium") is not None

if _has_gymnasium:
    import gymnasium
```

maybe we need to do this also in the rllib file